ztp/policygenerator/Makefile

```diff
@@ -0,0 +1,20 @@
+# Set kustomize env variable
```

Comment:
This comment isn't quite accurate, but I love how clean this makefile is now!

Reply:
ops, will remove it